docker/app
# [WIP] APP-214 Enable 'docker app validate' for experimental mode only

Signed-off-by: Anca Iordache <anca.iordache@docker.com>

Fixes  [APP-214](https://docker.atlassian.net/browse/APP-214)

Requires [PR-2095](https://github.com/docker/cli/pull/2095)

**- What I did**
Enabled validate command only for experimental mode

**- How I did it**
Added validate command to the list of commands only if experimental flag is on. 
**- How to verify it**
```
$ DOCKER_CLI_EXPERIMENTAL="disabled" docker app --help
Usage:	docker app COMMAND
Commands:
  ...
  run         Run an App from an App image
  update      Update a running App
```
```
$ DOCKER_CLI_EXPERIMENTAL="enabled" docker app --help
Usage:	docker app COMMAND
Commands:
  ...
  run         Run an App from an App image
  update      Update a running App
  validate    Check that an App definition (.dockerapp) is syntactically correct
```
```
$ DOCKER_CLI_EXPERIMENTAL="enabled" docker app validate
Validated "/home/anca/go/src/github.com/docker/app/examples/test.dockerapp"
```
```
$ DOCKER_CLI_EXPERIMENTAL="disabled" docker app validate
"validate" is not a docker app command
See 'docker app --help'
```
**- Description for the changelog**
Add validate to the list of docker app commands only in experimental mode.

**- A picture of a cute animal (not mandatory but encouraged)**
![8d1](https://user-images.githubusercontent.com/809903/65347491-67f7e500-dbdf-11e9-9594-319b8d9251ca.jpg)
